client/lib/post-formats/test/store.js

```diff
@@ -2,8 +2,7 @@
  * External dependencies
  */
 var expect = require( 'chai' ).expect,
-	sinon = require( 'sinon' ),
-	rewire = require( 'rewire' );
+	sinon = require( 'sinon' );
```

Comment:
I cleaned up imports. I don't think `useSandbox` makes sense here. W only use spy to do some weird tricks around handler. Hopefully we move everything to Redux soon and forget about those hacks :) In my opinion we should deprecate `use-sinon` helper and promote usage of [sinon.test](http://sinonjs.org/docs/#sinon-test). It has also sort of parent version [sinon.testCase](http://sinonjs.org/docs/#sinon-testCase). Together they do exactly the same what our helpers do :)